src/sentry/testutils/silo.py

```diff
@@ -134,7 +134,8 @@ def replacement_test_method(*args: Any, **kwargs: Any) -> None:
 
     def _call(self, decorated_obj: Any, stable: bool) -> Any:
         is_test_case_class = isinstance(decorated_obj, type) and issubclass(decorated_obj, TestCase)
-        is_function = callable(decorated_obj)
+        is_function = inspect.isfunction(decorated_obj)
```

Comment:
Hmm, looks subtle. Just curious, what's the distinction here?

Reply:
Technically, all classes are callable.  So we ended up with a scenario where class oriented pytest tests, were decorated as function pytest tests, which results in them not being collected (silently ignores the entire test suite)